src/vs/platform/userDataProfile/electron-main/userDataProfileStorageIpc.ts

```diff
@@ -66,7 +67,7 @@ export class ProfileStorageChangesListenerChannel extends Disposable implements
 		keys = keys.filter(key => key !== TARGET_KEY);
 		if (keys.length) {
 			const keyTargets = loadKeyTargets(this.storageMainService.applicationStorage.storage);
-			profileStorageValueChanges.push({ profile: this.userDataProfilesService.defaultProfile, changes: keys.map(key => ({ key, scope: StorageScope.PROFILE, target: keyTargets[key] })) });
+			profileStorageValueChanges.push({ profile: this.userDataProfilesService.defaultProfile, changes: keys.map(key => ({ key, scope: StorageScope.PROFILE, target: keyTargets[key], source: ValueChangeSource.PART })) });
```

Comment:
I miss a bit the context of this code (cc @sandy081), but is this truly `Part` and not `External`?